app/views/polls/questions/show.html.erb

```diff
@@ -55,11 +55,36 @@
   </div>
 </div>
 
+<% if @question.video_url.present? %>
+  <div class='row margin-top poll-question-show'>
+    <div class="small-12 medium-9 column video-link">
```

Comment:
You must move the `<div class="video-link">` inside the `small-12 medium-9 column` to align correctly this block. 👌 

![grid_freak](https://user-images.githubusercontent.com/631897/30126942-dffe1cc8-933d-11e7-8af1-690fc433007a.jpg)

Reply:
Nice one! I didn't notice 😅